R/diag_header.R

```diff
@@ -109,16 +109,16 @@ DATE_FIELD_NAME         <- "date"
 # -----------------------------------------------------------------------------
 # Name variables
 # Model base years
-model_base_years <- c( 1975, 1990, 2005, 2010 )
-X_model_base_years <- paste0( "X", model_base_years )
```

Comment:
I see that formatR hates the GCAM convention of putting extra spaces inside of parentheses and brackets.  I can't say I'm disappointed to see that. 😈